### PR TITLE
Jakarta Interceptors 1.2 apidocs

### PR DESCRIPTION
**Pull request by arjantijms**
Signed-off-by: arjantijms <arjan.tijms@gmail.com>

- [X] Specification JavaDoc in the wombat/x.y/apidocs directory.

### CODE REVIEW

File: interceptors/1.2/apidocs/allclasses-index.html [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+<li><a href="index-all.html">Index</a></li>
+<li><a href="help-doc.html">Help</a></li>
+</ul>
+</div>
+<a id="skip.navbar.bottom">
+<!--   -->
+</a>
+<!-- ======== END OF BOTTOM NAVBAR ======= -->
+</nav>
+<p class="legalCopy"><small>Copyright &#169; 1996-2013,     <a href="http://www.oracle.com">Oracle</a>     and/or its affiliates. All Rights Reserved.     Use is subject to     <a href="./doc-files/speclicense.html" target="_top">license terms</a>.</small></p>
```
Comment (gunnarmorling) [inline interceptors/1.2/apidocs/allclasses-index.html]: @arjantijms, the footer should refer to the EFSL and there should be no mention of Oracle as the copyright holder.

Comment (arjantijms) [reply interceptors/1.2/apidocs/allclasses-index.html]: You’re right, thx! I’m going through a series of all specs where this needs to be updated.

File: .project [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kwsutter) [inline .project]: I don't think we should be integrating the .project file...

Comment (arjantijms) [reply .project]: Sorry, not that one again. Second time this happens.

**Comment by dblevins** [comment]
@arjantijms mark this PR 'ready for review' when you get a chance.

**Pull request merged**